link/src/main/java/com/stripe/android/link/analytics/DefaultLinkEventsReporter.kt

```diff
@@ -34,12 +45,16 @@ internal class DefaultLinkEventsReporter @Inject constructor(
         fireEvent(LinkEvent.SignUpComplete, durationInSecondsFromStart(duration))
     }
 
-    override fun onSignupFailure(isInline: Boolean) {
-        fireEvent(LinkEvent.SignUpFailure)
+    override fun onSignupFailure(isInline: Boolean, error: Throwable) {
+        val params = mapOf(FIELD_ERROR_MESSAGE to (error.message ?: error.toString()))
```

Comment:
Is there any risk of us sending PII here?

Reply:
Yeah there is some risk the exceptions have sensitive data. Got a workaround.